src/indexer/migrations/20241009154948_create_tables.sql

```diff
@@ -0,0 +1,61 @@
+-- Add migration script here
+CREATE TABLE blocks (
+    hash BYTEA PRIMARY KEY,         -- Corresponds to BlockHash (inner Vec<u8>)
+    parent_hash BYTEA NOT NULL,     -- Parent block hash (BlockHash)
+    height BIGINT NOT NULL,         -- Corresponds to BlockHeight (u64)
+    timestamp TIMESTAMP NOT NULL,   -- UNIX timestamp (u64)
+    UNIQUE (height),                -- Ensure each block height is unique
+    CHECK (octet_length(hash) = 32) -- Ensure the hash is exactly 32 bytes
```

Comment:
Ça ça signifie qu'il faudra pad avec des 0 in front, ça me semble pas être une mauvaise idée dans l'absolu mais faudra le gérer dans le code